pkg/utils/ddp.go

```diff
@@ -47,8 +49,26 @@ type DDPpackage struct {
 	Name    string `json:"name"`
 }
 
+// 8 is an exit code of ddptool when profile was not found
+const ddpNoDDPProfile = 8
+
 var ddpExecCommand = exec.Command
 
+// IsDDPToolSupportedByDevice checks if DDPTool can be used with device
+func IsDDPToolSupportedByDevice(dev string) bool {
+	if _, err := GetDDPProfiles(dev); err != nil && !errors.Is(err, ErrProfileNameNotFound) {
```

Comment:
sorry I don't follow here why we check both the string and the existCode?

Reply:
As the error could imply that the DDP Tool is supported, but returns an empty package name, so it should return true for being supported. 
so we check the returned ExitCode with the ``ddpNoDDPProfile``